roles/bootstrap-os/defaults/main.yml

```diff
@@ -12,3 +12,11 @@ coreos_locksmithd_disable: false
 ## General
 # Set the hostname to inventory_hostname
 override_system_hostname: true
+
+## Downloads
+# Pypy version
+coreos_pypy_version: "7.0.0"
+# Pypy file name
+coreos_pypy_filename: "pypy3.5-{{ coreos_pypy_version }}-linux_x86_64-portable"
```

Comment:
Nit: That's the folder name, not the file name

Reply:
I' ve changed the references from filename to folder, and also added the variables to the README.md file in the role.